### PR TITLE
Add example instructions

### PR DESCRIPTION
**Pull request by rossmuir**
[Review on Reviewable](https://reviewable.io/reviews/dirvine/self_encryption/57)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,3 +46,28 @@ A version of [convergent encryption](http://en.wikipedia.org/wiki/Convergent_enc
 
 #Video of the process 
 [self_encryption process and use case video] (https://www.youtube.com/watch?v=Jnvwv4z17b4)
+
+#Basic self_encryptor example DIY instructions
+
+Instructions to use the 'basic_encryptor' example to encrypted a file into encrypted chunks and a datamap and then decrypted the datamap and restore the file.
```
Comment (Viv-Rajkumar) [inline README.md]: `to encrypted a file` should prolly be `to encrypt a file` similar for `decrypted` -> `decrypt`. Maybe give @NickLambert  a shout :)

Comment (rossmuir) [inline README.md]: Cheers @Viv-Rajkumar  updated

Comment (NickLambert) [inline README.md]: Suggest change from : Instructions to use the 'basic_encryptor' example to encrypted a file into encrypted chunks and a datamap and then decrypted the datamap and restore the file.

to 

Instructions to use the 'basic_encryptor' example. This library splits a file into encrypted chunks producing a data map, which is also split into chunks and encrypted.  The correct credentials decrypt the data map enabling the file to be reconstituted.

Comment (rossmuir) [inline README.md]: Cheers Nick I updated the PR with your feedback but omitted the "which is also split into chunks and encrypted" part as I don't believe the example does this part. Thanks :-)

**Comment by dirvine** [comment]
Nice one Ross, thx very much. will help community a lot.

**Pull request merged**